packages/apps/human-app/frontend/src/shared/helpers/get-chip-status-color.ts

```diff
@@ -0,0 +1,16 @@
+import { type ColorPalette } from '@/styles/color-palette';
+
+export function getChipStatusColor(status: string, colorPalette: ColorPalette) {
+  const normalizedStatus = status.toUpperCase();
+
+  switch (normalizedStatus) {
+    case 'ACTIVE':
```

Comment:
Why using hardcoded values and not values from enum here now (it was from enum in previous commits)?

This `getChipStatusColor` is placed in very common `shared/helpers`, but does every chip in app will have same set of statuses and colors? I suppose - no, so this utility takes place only for "jobs".

Let's do some kind of file/utility renaming, use enums instead of hardcoded values (it's their purpose)

Also not sure why we need to `const normalizedStatus = status.toUpperCase();` if we already do it while validating response schema

Reply:
We had a misunderstanding then :sweat_smile: We discussed these changes [here](https://github.com/humanprotocol/human-protocol/pull/2734#discussion_r1836818687) and since the proposed argument type was `string` I assumed that we will have to normalize it and compare strings as well; the `string` type also indicated that it will be used in places other than `jobs` so I left it there. All fixed in new commit!

Reply:
True, I didn't make myself clear, sorry

> the string type also indicated that it will be used in places other than jobs so I left it there.

I meant the general case if we had many different chips, but falling into the same set of rules in terms of color, which is hard to imagine tbh because they rely on status value that in general is specific for an entity (job in this case)

Thank you for double-checking it and pointing what was confusing!